app/assets/stylesheets/comments.scss

```diff
@@ -333,18 +333,24 @@ a.header-link {
       width: calc(100% - 80px);
       margin: 19px auto 8px;
       border: 1px solid #dbdbdb;
+      line-height: 2em;
```

Comment:
I believe this may not do the trick, will looks especially weird on longer paragraphs (more than one line). Increasing line-height won't actually increase the space between paragraphs that well.. Instead I'd suggest targeting all the children selectors and applying appropriate margin (similar to what we have in real comments), so:
```suggestion
      > * {
        margin: 4px 0px calc(1.1vw + 10px); // ok, i'm not a big fan of doing this weird math calc() but this is what we do for real comments sooo 🤷‍♂️
      }
```

Reply:
👍 on this, nice suggestion @ludwiczakpawel!

Reply:
Thanks @ludwiczakpawel, that's serious CSS mojo :D

Reply:
Works great, thank you!

![Screenshot_2020-03-19 All the King's Men Ad illo - DEV(local) Community 👩‍💻👨‍💻](https://user-images.githubusercontent.com/146201/77044647-14185e00-69c0-11ea-9df7-df4a935731e1.png)